File: src/main/java/org/occidere/githubnotifier/service/GithubUserService.java
package org.occidere.githubnotifier.service;

import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Service;

/**
 * @author occidere
 * @since 2019-11-29
 */
@Service
@RequiredArgsConstructor
public class GithubUserService {
    private final GithubUserRepository githubUserRepository;
}

File: src/main/scala/org/occidere/githubnotifier/service/GithubUserService.scala
package org.occidere.githubnotifier.service

import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Service

/**
 * @author occidere
 * @Blog: https://blog.naver.com/occidere
 * @Github: https://github.com/occidere
 * @since 2020. 08. 15.
 */
@Service
@Autowired
class GithubUserService(githubUserRepository: GithubUserRepository)
